solc/CommandLineInterface.cpp

```
@@ -887,6 +897,12 @@ void CommandLineInterface::handleAst()
 	}
 }
 
+void CommandLineInterface::serveLSP()
+{
+	lsp::JSONTransport transport;
+	lsp::LanguageServer{transport}.run();
```

Docs for `run()` say that it returns "boolean indicating normal or abnormal termination". We should return an error here or raise an exception if it returns `false`.